app/src/main/java/org/sopt/and/SignInAcitivity.kt

```diff
@@ -0,0 +1,205 @@
+                colors = TextFieldDefaults.colors(
+                    focusedContainerColor = Color.Gray,
+                    unfocusedContainerColor = Color.Gray
+                )
+            )
+
+            Box() {
+                TextField(
+                    value = passwordLogin,
+                    onValueChange = { passwordLogin = it },
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 20.dp),
+                    label = { Text("비밀번호") },
+                    colors = TextFieldDefaults.colors(
+                        focusedContainerColor = Color.Gray,
+                        unfocusedContainerColor = Color.Gray
+                    ),
+                    visualTransformation = if (isPasswowrdVisible) VisualTransformation.None else PasswordVisualTransformation()
+                )
+                TextButton(
+                    onClick = { isPasswowrdVisible = !isPasswowrdVisible },
+                    modifier = Modifier
+                        .padding(top = 20.dp, end = 8.dp)
+                        .align(Alignment.CenterEnd)
+                ) {
+                    Text(
+                        text = "show",
+                        color = Color.White,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+            }
+
+            Button(
+                onClick = {
+                    if (validateSignIn(email, password, emailLogin, passwordLogin)) {
+                        coroutineScope.launch {
+                            snackbarHostState.showSnackbar("로그인 성공!")
+                        }
+                        val intent = Intent (context, MyActivity::class.java)
+                        intent.apply {
+                            flags = Intent.FLAG_ACTIVITY_NEW_TASK
+                            putExtra("email",emailLogin)
+                        }
+                        context.startActivity(intent)
+                    } else {
+                        coroutineScope.launch {
+                            snackbarHostState.showSnackbar("로그인 실패!")
+                        }
+                    }
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 40.dp),
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = Color.Blue
+                )
+            ) {
+                Text(
+                    modifier = Modifier
+                        .padding(top = 10.dp, bottom = 10.dp),
+                    text = "로그인"
+                )
+            }
+        }
+    }
+
+
+}
+
+fun validateSignIn(
+    email: String,
+    password: String,
+    emailLogin: String,
+    passwordLogin: String
+): Boolean {
+    if (email.equals(emailLogin) && password.equals(passwordLogin)) {
+        return true
+    } else {
+        return false
+    }
+}
```

Comment:
```suggestion
): Boolean = email.equals(emailLogin) && password.equals(passwordLogin)
```
이런 식으로 한 번에 작성할 수도 있을 것 같아요!
결과 값이 Boolean이니까요.